### DISCUSSION TITLE
Custom Reports based on the user details such as email

### DISCUSSION THREAD

**Discussion by ganeshmani** [electerious/Ackee#275] [Q&A]
This is an Awesome library. Thank you so much for creating it. I would like to store the page views based on the user email. For example, When a protected page loads, i will ask for the user email. When they fill that, i need to track the page views, average time spent per user(email) etc. 

Is there any way to achieve this with Ackee? Thanks in advance.

**Comment by electerious** [comment]
It should be possible via the `ackee-tracker` API. You could either use the Events API to build a custom solution or you could attach the email to the `siteLocation`. It uses `window.location.href` by default, but you can send anything that is a valid URL: https://github.com/electerious/ackee-tracker#recorddomainid-attributes-callback
